lib/rules/await-fire-event.ts

```diff
@@ -10,8 +10,13 @@ import {
 } from '../node-utils';
 
 export const RULE_NAME = 'await-fire-event';
+const EVENT_MODULES = ['fireEvent', 'userEvent'] as const;
```

Comment:
We have `EVENTS_SIMULATORS` exported in `utils`. Any problem with reusing that one?